docs/src/main/paradox/s3.md

```diff
@@ -128,6 +128,50 @@ Scala
 Java
 : @@snip [snip](/s3/src/test/java/docs/javadsl/S3Test.java) { #list-bucket-attributes }
 
+
+## Bucket management
+
+Bucket management API provides functionality for both Sources and Futures / CompletionStages. 
+In case of the Future API user can specify attributes to the request in the method itself and as for Sources it can be done via method `.withAttributes`.
+For more information about attributes see: @scaladoc[S3Attributes](akka.stream.alpakka.s3.S3Attributes$) and @scaladoc[Attributes](akka.stream.Attributes)
+
+### Make bucket
+In order to create a bucket in S3 you need to specify it's unique name. This value has to be set accordingly to the [requirements](https://docs.aws.amazon.com/awscloudtrail/latest/userguide/cloudtrail-s3-bucket-naming-requirements.html).
```

Comment:
The link seems to be the wrong one (cloudtrail), guess this is the right one ? -> https://docs.aws.amazon.com/AmazonS3/latest/dev/BucketRestrictions.html#bucketnamingrules

Reply:
Yes, thank you for noticing. My bad on not checking it before merging. I will submit another PR tomorrow referencing that issue